e2e_tests/azure_function/test_azure_function_deployment.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import subprocess
+import time
+import uuid
+
+import requests
+
+from e2e_tests.cli_operations import delete_deployment
+
+logger = logging.getLogger('bentoml.test')
+
+
+def test_azure_function_deployment(iris_clf_service):
+    random_hash = uuid.uuid4().hex[:6]
+    deployment_name = f'test-azure-{random_hash}'
+    create_deployment_command = [
```

Comment:
here it might be a lot easier to read and write with f string, e.g.

create_deployment_command = f"bentoml azure-functions deploy {deployment_name} .."